LoggingKata/TacoParser.cs

```diff
@@ -9,10 +11,78 @@ public class TacoParser
         
         public ITrackable Parse(string line)
         {
-            logger.LogInfo("Begin parsing");
+
+            string[] cells = line.Split(',');
+
+            if (cells.Length < 3)
+            {
+                logger.LogInfo("Less than 3");
+                return null;
+              
+            }
+
+            double Long;
+            double Lat;
+
+
+            if (Double.TryParse(cells[0], out Lat) && Double.TryParse(cells[1], out Long)) 
+                {
+
+                Taco taco = new Taco();
+                taco.Name = cells[2];
+
+                taco.Location = new Point { Latitude = Lat, Longitude = Long };
+
+                return taco;
+
+                }
+            else
+            {
+                Console.WriteLine(cells[2]);
+                logger.LogInfo("Failed to Parse");
+                return null;
+            }
+
+
+//
+
+
+
+
+            // using the method 
 
             // Do not fail if one record parsing fails, return null
-            return null; // TODO Implement
+            //return null; // TODO Implement
+
+
+
+            // Take your line and use line.Split(',') to split it up into an array of strings, separated by the char ','
```

Comment:
These comments were probably helpful while getting the method to work, but now they should be removed, especially since they're in the readme file as well